src/reports/components/report-sections/minimal-rule-header.tsx

```diff
@@ -36,8 +37,9 @@ export const MinimalRuleHeader = NamedFC<MinimalRuleHeaderProps>('MinimalRuleHea
 
     return (
         <span className="rule-detail">
+            <span className={outcomeChipContainer}>{renderCountBadge()}</span>
             <span>
-                {renderCountBadge()} {renderRuleName()}: {renderDescription()}
```

Comment:
Should the renderCountBadge function also be deleted? This seems to be the only place where it's called

Reply:
hey @DaveTryon, we're still using `renderCountBadge` we just move it to a different place inside the same component.